projects/uitdatabank/docs/terms.md

```diff
@@ -1,7 +1,95 @@
 # Terms
 
-This page is still under construction. For now the documentation about terms can be found on the old documentation portal:
+The categorisation model of UiTdatabank is a standard aimed at simplifying data exchange and avoiding complex mapping procedures.
 
-* [Event types](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/type_aanbod.html)
-* [Event themes](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/combinatie_type_thema.html)
-* [Facilities](https://documentatie.uitdatabank.be/content/uitdatabank/latest/categorisatie/voorzieningen.html)
+The main purpose of the categorisation model is to categorise cultural and leisure activities. The categorisation model consists of 3 dimensions (`domains`):
+1. `eventtype`
+2. `theme`
+3. `facility`
+
+There is no strong hierarchy between the different dimensions: there is only a hierarchical relationship between the first and second dimension.
+
+The complete categorisation model is available in our [Taxonomy API](https://taxonomy.uitdatabank.be/terms).
```

Comment:
```suggestion
The complete categorisation model is available in our [Taxonomy API](https://docs.publiq.be/docs/uitdatabank/4047c82490760-list-taxonomy-terms).
```

Preferably we always link to documentation of an endpoint instead of the endpoint itself. In most cases it requires authentication anyway. It just so happens that this endpoint does not, which technically makes it possible to link to it. But the documentation gives extra info that currently cannot be found by visiting the endpoint itself.